lib/js/styles/ColorsList.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+	<div>
+		<div v-for="colorList in colors" :key="colorList.title" class="colorDefinition">
+			<div class="colorsList" :class="{ [colorList.class]: true }">
```

Comment:
when class is not defined it generates `undefined`

Reply:
Fixed.